lib/postgrex/extensions/binary.ex

```diff
@@ -373,6 +382,23 @@ defmodule Postgrex.Extensions.Binary do
     <<value_byte_size::int32>> <> value
   end
 
+  defp encode_network(%Postgrex.INET{address: {_, _, _, _} = addr}),
```

Comment:
The tuple match is not needed here because we match in `tuple_to_binary` anyway.